magda-web-client/src/Components/FeedbackForm.js

```diff
@@ -0,0 +1,120 @@
+import React from 'react';
+import Input from 'muicss/lib/react/input';
+import {fetchFeedback, resetFeedback} from '../actions/feedbackActions';
+import { connect } from 'react-redux';
+import { bindActionCreators } from 'redux';
+import './FeedbackForm.css';
+import Button from 'muicss/lib/react/button';
+import Textarea from 'muicss/lib/react/textarea';
+import feedback from "../assets/feedback.svg";
+import close from "../assets/close.svg";
+import success from "../assets/success.svg";
+import Notification from '../UI/Notification';
+
+
+class FeedbackForm extends React.Component {
+  constructor(props) {
+    super(props);
+    this.state = {
+      isOpen: false,
+      name: '',
+      email: '',
+      feedback: ''
+    };
+    this.onCancel = this.onCancel.bind(this);
+    this.changeValue = this.changeValue.bind(this);
+    this.onSubmit = this.onSubmit.bind(this);
+    this.onDismissNotification = this.onDismissNotification.bind(this);
+  }
+
+  onSubmit(){
+    this.props.fetchFeedback(JSON.stringify({
+          name: this.state.name,
+          email: this.state.name,
+          feedback: this.state.feedback
```

Comment:
It seems feedback API expects the data comes in with key `comment ` rather than ``feedback
See:
https://github.com/TerriaJS/magda/blob/35e83571d99f58f86d62ae3469914a9edc8130ec/magda-feedback-api/src/index.ts#L91

Reply:
Good spotting!